Xamarin.Forms.Platform.UAP/StepperRenderer.cs

```diff
@@ -80,5 +80,21 @@ void UpdateValue()
 		{
 			Control.Value = Element.Value;
 		}
+
+		bool _isDisposed;
```

Comment:
Could we move this variable definition with the rest of the variables?

Reply:
Done